test/Conversion/TorchOnnxToTorch/simple_ops_g_to_p.mlir

```diff
@@ -160,6 +162,71 @@ func.func @test_matmul_4d(%arg0: !torch.vtensor<[1,2,3,4],f32>, %arg1: !torch.vt
 
 // -----
 
```

Comment:
Overall try to make the style of the lit tests match the previous tests above. We try to make these easy to read as failures become easier to understand.